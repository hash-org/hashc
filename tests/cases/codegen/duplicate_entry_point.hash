// stage=full, run=fail

main := () => {

}

main := () => {

}

#entry_point my_main := () => {

}
