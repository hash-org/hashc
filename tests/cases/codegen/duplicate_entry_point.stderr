error[0007]: multiple entry points declared
  --> $DIR/duplicate_entry_point.hash:7:9
5  |    }
6  |    
7  |    main := () => {
   |  __________-
8  | |  
9  | |  }
   | |__- cannot declare another entry point
10 |    
11 |    #entry_point my_main := () => {

  --> $DIR/duplicate_entry_point.hash:3:9
1  |    // stage=full, run=fail
2  |    
3  |    main := () => {
   |  __________-
4  | |  
5  | |  }
   | |__- the entry point is already declared here
6  |    
7  |    main := () => {
